docs/getting-started/installation/proxmox.mdx

```diff
@@ -0,0 +1,67 @@
+# Proxmox
+
+This guide covers how to install homarr inside of a linux-container in ProxmoxVE, using a fully automated approach.
+
+# What is community-scripts?
+
+https://community-scripts.github.io/ProxmoxVE/
+
+We are a community-driven initiative that simplifies the setup of Proxmox Virtual Environment (VE).
+With hundreds of scripts to help you manage your Proxmox VE environment. Whether you're a seasoned user or a newcomer, we've got you covered.
+
```

Comment:
Is is not clear how the scripts community is related to this script. Therefore, I suggest the following:
```suggestion
This installation method is provided by the Proxmox Community Scripts. We are a community-driven initiative that simplifies the setup of Proxmox Virtual Environment (VE).
With hundreds of scripts to help you manage your Proxmox VE environment. Whether you're a seasoned user or a newcomer, we've got you covered.

```